src/design_docs/org/lib/app.coffee

```diff
@@ -1,22 +1,38 @@
 _ = require('./underscore')
 h = require('./helpers')
-validation = require('./validation/validate')
+validate = require('./validation/validate')
 actions = require('./actions')
 audit = require('./shared/audit')
 
-dd =
-  views: {}
+auth = validate.auth
 
+dd =
+  views:
+    by_role:
+      map: (doc) ->
+        validate = require('views/lib/validation/validate')
+        if not validate._is_team(doc)
+          return
+        team_id = doc._id.slice(5)
```

Comment:
Is slice the best option here or might something like replace or another function be more readable? I happen to know that you're replacing 'team_' but not obvious necessarily in the .coffee file.